### ISSUE TITLE
Midi file playback fails using AU plugins under Rosetta, Possibly native intel macs.

### ISSUE THREAD

**Issue by arfy8820** [losnoco/Cog#332]
Hello, when attempting to play midi files using an AU plugin, for example, Roland's sound canvas VA, playback simply fails under Rosetta. However, it does work if cog is run natively. More seriously, however, playback with plugins could be entirely broken for those running on intel macs. A friend of mine has an intel mac while traveling, and said that plugins just failed for him too.
More than happy to provide more info/debug logs if needed.

**Comment by arfy8820** [comment]
Understood. I'll look through my collection and see if any free ones are worth testing.

**Comment by kode54** [comment]
Well, that was a horrible waste of $9.99.

    *Sound Canvas VA is supported on macOS 10.11-15 (Intel only) and macOS 11 (Intel only). Sound Canvas VA is not supported with macOS 12, macOS 13, and Apple silicon. Please consider this before purchasing.

**Comment by lumpython** [comment]
Interesting, because Sound Canvas VA runs on my Apple Silicon Mac, I can hear the sound, but I don't see the AU interface, is anyway to show the instrument interface? ![image](https://github.com/losnoco/Cog/assets/50165399/0cd1ccfc-d497-42ab-a52c-094314005187)

**Comment by lumpython** [comment]
Interesting, because Sound Canvas VA runs on my Apple Silicon Mac, I can hear the sound, but I don't see the AU interface, is anyway to show the interface?![image](https://github.com/losnoco/Cog/assets/50165399/513187fe-2f6a-43f7-9c85-7a243ac5fc9e)

**Comment by kode54** [comment]
Yes, it seems to work now. I could make it show the instrument interface, but the interface would be in its own window, and there would be three of them at a time, per file you listen to, and they'd close again on every track change. Also, anything they show relative to playback would occur up to 2-10 seconds ahead of what you hear, because the instrument interfaces are designed for real time playback, and I'm essentially using them in offline rendering mode.

**Comment by lumpython** [comment]
I also noticed midi seems don't sound correctly, instruments mapped to Piano except percussion, it could be SoundCanvas's bug because I encountered same problem on SoundCanvas VA on iOS after a major iOS update ages ago, it just doesn't sound as same as I remember unless if can check SoundCanvas panel as it shows instrument info and settings. Roland's VSTs seem has serious latency issue on Retina Display anyway, I found it runs better on older Macs.

**Comment by kode54** [comment]
This seems to be fixed now, I think? Please report.
